### PR TITLE
Add a note about testing to the readme

### PR DESCRIPTION
**Pull request by orf**
As per #46

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -843,6 +843,15 @@ Managing the permissions rule set
     Tests the rule with the given name. See ``RuleSet.test_rule``.
 
 
+Testing
+=======
+
+When using Django's built in `TestClient` using the `force_login` method will fail unless you specify a specific authentication backend.
+
+To do this pass ``django.contrib.auth.backends.ModelBackend`` as the ``backend`` argument:
```
Comment (blueyed) [inline README.rst]: s/To do/To work around/

**Comment by orf** [comment]
Updated :+1:

**Comment by blueyed** [comment]
Should be updated to mention that it will be fixed in the next release probably.

**Comment by orf** [comment]
I'd rather leave it out until an actual release comes out with the fix, but I added a speculative 'up to Django 1.10' sentence. Is that OK @blueyed / @dfunckt ?

**Comment by dfunckt** [comment]
Included in c560b964, thanks!

**Pull request closed without merging**